particles/tlv/particle.py

```diff
@@ -7,14 +7,13 @@
 import math
 
 class Particle(BaseParticle, RootObj):
-    def __init__(self, pdgid, charge, tlv, status=1, index=None, subtype='u'):
+    def __init__(self, pdgid, charge, tlv, status=1):
         super(Particle, self).__init__()
         self._pid = pdgid
-        self._uid = 0
-        if index != None:
-            self._uid = Identifier.make_id(Identifier.PFOBJECTTYPE.PARTICLE, index, subtype, tlv.E())
+        self._dagid = None
```

Comment:
Set _dagid to None in BaseParticle, and remove here? (and in FCC class)

Reply:
OK